src/System.Private.CoreLib/shared/System/StringComparer.cs

```diff
@@ -146,7 +146,7 @@ public new bool Equals(object? x, object? y)
             return x.Equals(y);
         }
 
-        public int GetHashCode(object obj)
+        public int GetHashCode(object? obj) // TODO-NULLABLE: https://github.com/dotnet/roslyn/issues/23268
```

Comment:
Instead of annotating it incorrectly, can we annotate it correctly and use a #pragma to suppress the warning, with the pragma appropriately commented?

Reply:
Makes sense.